src/js/plugins/inheritance.js

```diff
@@ -1632,7 +1632,34 @@
       }
     },
 
+    /*
+     * The init function is the first code the chrome plugin will run before 
+     * doing anything.
+     */
     init: function () {
+      /*
+       * The following code checks if a any chagne is made to the url indicating
+       * a card is open. If the url changed then we rerender the card with the 
+       * powerups added by calling updateCardView.
+       */
+      // store url on load
+      var currentPage = window.location.href;
+
+      // listen for changes
+      setInterval(function () {
+          if (currentPage != window.location.href)
+          {
+              // page has changed, set new page as 'current'
+              currentPage = window.location.href;
+
+              // update only if a card is open (which we can tell by checking if there is a /c/ in the url)
+              if (currentPage.includes("/c/"))
+              {
+                self.updateCardView();
+              }
+          }
+      }, 100);
```

Comment:
See if you can find a way to detect url change without having to poll every 100ms

Reply:
Surprisingly, it looks like reliable ways to detect the URL change are in short supply. https://stackoverflow.com/questions/6390341/how-to-detect-url-change-in-javascript